src/components/ResultMap.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState } from "react";
+import { Map, GoogleApiWrapper, Marker, InfoWindow } from "google-maps-react";
+import { connect } from "react-redux";
+
+const ResultMap = props => {
+  const [showInfoWindow, setShowInfoWindow] = useState(false);
+  const [activeMarker, setActiveMarker] = useState({});
+  const [selectedPlaces, setSelectedPlaces] = useState({});
+
+  let activityTypes;
+  let marker
+  let activityMarkers = [];
+
+  useEffect(() => {}, [props.activities]);
```

Comment:
what does one do?

Reply:
Ah, nothing anymore after setActivities was moved to another component. Removed.